src/clojure/nrepl/server.clj

```diff
@@ -158,9 +158,25 @@
       (binding [dynamic-loader/*state* state]
         ((:handler @state) msg)))))
 
-(defrecord Server [server-socket port open-transports transport greeting handler]
-  java.io.Closeable
-  (close [this] (stop-server this)))
+(defrecord 
+ Server
+  ;;A record representing an nREPL server.
```

Comment:
It seems to me this comment is off by one space.

Reply:
I think we've now satisfied the formatter :)